doc/stdenv/meta.chapter.md

```diff
@@ -134,7 +134,29 @@ Attribute Set `lib.platforms` defines [various common lists](https://github.com/
 This attribute is special in that it is not actually under the `meta` attribute set but rather under the `passthru` attribute set. This is due to how `meta` attributes work, and the fact that they are supposed to contain only metadata, not derivations.
 :::
 
-An attribute set with as values tests. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise. A derivation that is a test needs to have `meta.timeout` defined.
+An attribute set with tests as values. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise. A derivation that is a test needs to have `meta.timeout` defined.
+
+#### Regular tests
+
+Tests that are part of the source package are commonly executed in the `installCheckPhase`. For tests that are introduced in nixpkgs, adding them to `passthru.tests` is preferable because `installCheckPhase` introduces overhead for every build, where `passthru.tests` can be executed independently of the build.
+
+```nix
+{ /* ... */ }:
+let
+  key = mkDerivation {
```

Comment:
Does `${version}` require this?

```suggestion
  key = mkDerivation rec {
```

Reply:
I guess this question is too focused on the trees; my forest-thought was whether anything in the example might mislead new packagers.